testing/regress/ecl/indexmerge.ecl

```diff
@@ -0,0 +1,40 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=file
+//class=index
+//version multiPart=false
+//version multiPart=true
+
+import ^ as root;
+multiPart := #IFDEFINED(root.multiPart, true);
+useLocal := #IFDEFINED(root.useLocal, false);
+useTranslation := #IFDEFINED(root.useTranslation, false);
+
+//--- end of version configuration ---
+
+import $.setup;
+import setup.TS;
+Files := setup.Files(multiPart, useLocal, useTranslation);
+
+//Read from an index, merging from a large number of cursor positions to generate a sorted output
+
+TS_searchIndex := Files.getSearchIndex();
+filtered := TS_searchIndex(KEYED(kind=1 AND word[1]='a'));
+withOrder := STEPPED(filtered, doc, PRIORITY(3),HINT(maxseeklookahead(50)));
+
+OUTPUT(COUNT(NOFOLD(withOrder)));
```

Comment:
I think it would be better if we expect the "Pass" as output instead of that cryptic/magic number of 305475. 
If the count doesn't match, then we can get an output something like "Failed: result is x expected value is 305475"